wiki/markdown/rules.md

```diff
@@ -95,11 +95,10 @@ This field denotes if this event type starts the object aggregation. If
 StartEvent is "YES" then it will be the first processed event in the aggregation
 sequence. If StartEvent is "NO" then Eiffel events of this type will be used to
 append information to existing aggregated object. If no aggregated object exist
-then it will wait a certain time as defined by property
-_waitlist.collection.ttl_ in [application's properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
+then it will wait a certain time as defined by the property [_waitlist.collection.ttl_](configuration.md#waitlist)
```

Comment:
Please remove the italic style on the link. I find it harder to read and I think the link itself adds emphasis enough to the word.

Reply:
Done